src/site/stages/build/drupal/graphql/facilities-fragments/healthCareRegionHealthServices.node.graphql.js

```diff
@@ -86,7 +87,7 @@ module.exports = `
     ${HEALTH_SERVICES_RESULTS}
   }
   primaryCareHealthServices: queryFieldClinicalHealthServices(${queryFilter(
-    'Primary care (Family medicine, internal medicine)',
+    'Primary care',
```

Comment:
Is there an id or non-user facing string we could use to avoid having to update these when the title changes?

Reply:
yes. i want to propose a different architecture for this all together but this is needed now. definitely don't want to hard code this :sweat: